js/controller.js

```diff
@@ -28,7 +28,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
   var Html5Skin = function (mb, id) {
     this.mb = mb;
     this.id = id;
-    this.videoVrSource = {};
+    this.videoVrSource = false;
```

Comment:
Would null work better here? (if videoVrSource is intended to be an object)

Reply:
+